egs/jsut/tts1/run.sh

```diff
@@ -141,7 +141,7 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
 fi
 
 if [ -z ${tag} ]; then
-    expname=${train_set}_${backend}_$(basename ${train_config%.*})_${trans_type}
+    expname=${train_set}_${backend}_$(basename ${train_config%.*})
```

Comment:
Is it intentional? I thought that adding trans_type here is convenient.

Reply:
I added `${trans_type}` to the head of `${train_set}`.
See this commit.
https://github.com/espnet/espnet/pull/1258/commits/c83d31e051b2c059f1b65848c70086c2ba846b76

Reply:
Oh I see. Thanks!